itis.playground/Pages/ThirdTask.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,296 @@
+//: [Previous](@previous)
+
+import Foundation
+
+protocol Student {
+    var name: String {get set}
```

Comment:
```suggestion
    var name: String { get }
```
1. стиль 
2. ну и тут наверное достаточно возвращать значение, потому что имя не меняется

Reply:
везед стиль поправь